docs/UserGuide.md

```diff
@@ -544,21 +525,53 @@ Listed here are some frequently asked questions. Should you require any other as
 **Q**: Why are there no guests in my GuestBook?<br>
 **A**: If you didn't enter the `clear` command, it is likely that your `guestbook.json` file is **corrupted**, causing GuestBook to reset it to prevent any malicious files from being used.
 
+**Q**: Why am I getting error messages for fields that are correct, or I'm not even editing?<br>
+**A**: If you are using the `add`/`edit` commands and input the `name`/`request` fields, ensure that they do not contain any field prefixes after whitespace in their content (e.g., `rq/Extra bed & p/c`), as GuestBook could be parsing part of your input as another field. Consider using capital letters as this check is case-sensitive (`rq/Extra bed & P/C` will be parsed correctly). You may refer to [Field summary](#field-summary) for more information about field constraints.
+
+[Return to Table of Contents](#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------
+
+### Field summary
+
+The format and constraints of fields in GuestBook are listed in the table below.
+
+In the case where GuestBook rejects your field inputs, you can refer to this section to use the right format.
+
+Please note that if you are specifying the `name`/`request` fields, ensure that they do not contain any field prefixes after whitespace in their content (e.g., `rq/Extra bed & p/c`), as GuestBook could be parsing part of your input as another field.
```

Comment:
For this note, should we make this into a callout of the information type?

Reply:
Should we not mention how GuestBook parses the field as it might seem overly technical for our target audience? Maybe something like "... as GuestBook may mistaken your input as another field and may not function as you expect"?

Reply:
Agreed, changed!